src/main/java/discord4j/discordjson/json/PartialUserData.java

```diff
@@ -20,8 +20,16 @@ static ImmutablePartialUserData.Builder builder() {
 
     Id id();
 
+    @JsonProperty("global_name")
+    Possible<String> globalName();
+
     Possible<String> username();
 
+    /**
+     * @deprecated For removal
+     * @return "0000" if the user is using the new username system, otherwise the discriminator
+     */
+    @Deprecated
```

Comment:
hmm maybe use the for-removal here?
```suggestion
    @Deprecated(forRemoval = true)
```

Reply:
The project is targeting java 8, and this feature requires java 9 :(

Reply:
Oh i forget check that xd